src/app/pagenotfound/pagenotfound.component.scss

```diff
@@ -0,0 +1,44 @@
+@use '@angular/material' as mat;
+@import '/src/materialColorScheme.scss';
+
+.about-dockstore {
+  margin-top: 6rem;
+  background-image: url(../../assets/images/about-page/city.png);
+  background-repeat: repeat-x;
+  background-position-y: bottom;
+}
+
+.truck-img {
```

Comment:
Seems like these are duplicated from the about page scss? If so, I would be inclined to move them to the global sheet with an explanation of what these are for (text page with truck background, for example)